src/figuresClasses.ts

```diff
@@ -1,11 +1,93 @@
-export interface Figure {}
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
-export class Triangle implements Figure {}
+export class Triangle implements Figure {
+  readonly shape: Shape = Shape.Triangle;
```

Comment:
Here and in other places - you may skip some types, as TypeScript is smart enough to guess them

```suggestion
  readonly shape = Shape.Triangle;
```